setup.cfg

```diff
@@ -95,13 +96,14 @@ all =
     typing_extensions>=3.10.0.1
     fsspec[http]>=2022.8.2
     s3fs>=2022.8.2
+    timezonefinder
```

Comment:
Where did this come from?

Reply:
Ah, so this is the cost of enabling that coordinates test above. Hmm... I guess it is ok for now. But if it causes too much maintenance headache in the future, we can revisit this.

Reply:
Will do!

Reply:
BTW, in case you missed the Slack msg, I will fold this into my edit and I plan to push to this branch directly, so pls don't touch it or you might lose your changes. FYI. Thanks!